profile.json.example

```diff
@@ -1,6 +1,7 @@
 {
   "user": "user name",
-  "account": "account name",
+  "url": "https://account_name.snowflakecomputing.com:443"
```

Comment:
Nit, I think we can drop the 443

Reply:
At least in the KC configuration doc https://docs.snowflake.com/en/user-guide/kafka-connector-install.html, the url in the example contains a port number, is it possible that we use a different port?